src/app/compare/selectors.ts

```diff
@@ -45,14 +46,20 @@ export const compareItemsSelector = createSelector(
   }
 );
 
+const organizerTypes = [
+  ItemCategoryHashes.Armor,
+  ItemCategoryHashes.Weapon,
+  ItemCategoryHashes.Ghost,
+];
+
 /**
  * Returns a link to the organizer for the current compare search.
  */
 export const compareOrganizerLinkSelector = createSelector(
   currentAccountSelector,
   compareSessionSelector,
   (account, session) => {
-    if (!session || !account) {
+    if (!session || !account || organizerTypes.includes(session.itemCategoryHashes[0])) {
```

Comment:
maybe i am spacing out here but it really feels like this should be `!organizerTypes.includes`??

it seems to be suppressing the button from appearing in weapon compare sheets. although weirdly for not armor or ghosts.

Reply:
That's right, I'll fix it